DEVELOPMENT.md

```diff
@@ -76,7 +76,8 @@ While iterating on the project, you may need to:
 
 1. [Install Tekton pipeline](#install-pipeline)
 2. [Building Tekton client](#building-tekton-client)
-1. [Add and run tests](./test/README.md#tests)
+3. [Add and run tests](./test/README.md#tests)
```

Comment:
fyi: markdown simplifies ordered list e.g. 

```md
1. foo 
1. bar
1. baz
```

Reply:
oh nice, I was not aware of this, so corrected the order.